packages/ai-constructs/API.md

```diff
@@ -103,10 +98,18 @@ declare namespace runtime {
         ConversationMessageContentBlock,
         ConversationTurnEvent,
         ExecutableTool,
-        handleConversationTurnEvent
+        handleConversationTurnEvent,
+        ToolDefinition
     }
 }
 
+// @public (undocumented)
+type ToolDefinition = {
+    name: string;
+    description: string;
+    inputSchema: ToolInputSchema;
+};
```

Comment:
This abstraction appeared in third place already in public API. Therefore, extracting a type.